Documentation/ApiOverview/CachingFramework/Developer/Index.rst

```diff
@@ -1,143 +1,165 @@
-.. include:: /Includes.rst.txt
+..  include:: /Includes.rst.txt
 
-
-
-.. _caching-developer:
+..  _caching-developer:
 
 =====================
 Developer Information
 =====================
 
-This chapter is targeted at extension authors who want to use the caching framework
-for their needs. It is about how to use the framework properly. For details about
-its inner working, please refer to the :ref:`section about architecture <caching-architecture>`.
+This chapter is targeted at extension authors who want to use the caching
+framework for their needs. It is about how to use the framework properly. For
+details about its inner working, please refer to the
+:ref:`section about architecture <caching-architecture>`.
 
-Example usages can be found throughout the TYPO3 CMS Core, in particular in
-system extension `core` and `extbase`.
+Example usages can be found throughout the TYPO3 Core, in particular in system
+extensions `core` and `extbase`.
 
 
-.. _caching-developer-usage:
-.. _caching-developer-registration:
+..  _caching-developer-usage:
+..  _caching-developer-registration:
 
 Cache Registration
 ==================
 
-Registration of a new cache should be done in :file:`ext_localconf.php`. The example below just defines
-an empty sub-array in *cacheConfigurations*. Neither *frontend* nor *backend* are defined,
-meaning that the cache manager will choose the default :ref:`variable frontend <caching-frontend-variable>`
-and the :ref:`database backend <caching-backend-db>` by default.
-
-.. code-block:: php
+Registration of a new cache should be done in :file:`ext_localconf.php`. The
+example below just defines an empty sub-array in *cacheConfigurations*. Neither
+*frontend* nor *backend* are defined, meaning that the cache manager will choose
+the default :ref:`variable frontend <caching-frontend-variable>` and the
+:ref:`database backend <caching-backend-db>` by default.
 
-   $GLOBALS['TYPO3_CONF_VARS']['SYS']['caching']['cacheConfigurations']['myext_mycache'] ??= [];
+..  code-block:: php
+    :caption: EXT:some_extension/ext_localconf.php
```

Comment:
```suggestion
    :caption: EXT:my_extension/ext_localconf.php
```
Should the extension and the array key be alike? EXT:my_extension -> myext_mycache?

Reply:
No, below is some_extension already used. So I chose to use it also here.

Reply:
Done